src/libraries/System.Runtime/tests/System/ArrayTests.cs

```diff
@@ -4746,7 +4746,14 @@ public static void Copy_LargeMultiDimensionalArray()
             // If this test is run in a 32-bit process, the large allocation will fail.
             if (IntPtr.Size != sizeof(long))
             {
-                return;
+                throw new SkipTestException("Unable to allocate enough memory");
+            }
+
+            if (PlatformDetection.IsUbuntu1804 || PlatformDetection.IsSLES)
+            {
+                // On these platforms, occasionally the OOM Killer will terminate the
+                // tests when they're using ~1GB, before they complete.
+                throw new SkipTestException("Unable to allocate enough memory");
```

Comment:
This message isn't correct.

Reply:
Well in my mind if I allocate the memory then promptly get terminated for it, .. But I updated the string.